myproject1/app1/tests.py

```diff
@@ -0,0 +1,39 @@
+import unittest
+import os
+
+from django.test import TestCase,Client
+from django.core.mail import send_mail
+from django.contrib.auth.models import User
+from django.core.urlresolvers import reverse
+from django.test import RequestFactory
+
+from . import views
+from .forms import SignupForm,LoginForm
+
+# Create your tests here.
+"""
+def setUp(self):
+
+    self.user = UserFactory()
+    self.factory = RequestFactory()
+
+"""
+class Test1(unittest.TestCase):
+	
```

Comment:
As discussed, 
- please dont use hardcoded values for the test cases. 
- please work on view test cases as they are also important.
- Discuss the test scenarios before implementing them.